.travis.yml

```diff
@@ -51,7 +51,7 @@ before_script:
 
 # Install composer
   - composer validate
-  - composer require --prefer-dist --no-update silverstripe/recipe-testing:^1 silverstripe/recipe-cms:4.2.x-dev
+  - composer require --prefer-dist --no-update silverstripe/recipe-testing:^1 silverstripe/recipe-cms:4.3.x-dev
   - if [[ $DB == PGSQL ]]; then composer require --prefer-dist --no-update silverstripe/postgresql:2.0.x-dev; fi
```

Comment:
Can you bump the postgres version to 2.1.x-dev here too please? If I do it in a separate commit it seems to cause merge conflicts with your PRs =(

Reply:
But I can do it afterwards! Behat failure is pre-existing too